rsconf/component/sirepo_sandbox.py

```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this should call this devbox. sirepo_sandbox is tool long. The instances would be devbox_ecarlin, devops_nagler, etc.

Reply:
aeab41b